src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -0,0 +1,255 @@
+    }
+
+    public Command driveNormal() {
+        // The command is named "Sample Command" and requires this subsystem.
+        return new Command("driveNormal", this) {
+            @Override
+            protected void initialize() {
+                // Called just before this Command runs the first time
+            }
+
+            @Override
+            protected void execute() {
+                // Called repeatedly when this Command is scheduled to run
+                drive.arcadeDrive(
+                        Robot.driveController.getTriggerAxis(Hand.kRight)
+                                - Robot.driveController.getTriggerAxis(Hand.kLeft),
+                        Robot.driveController.getX(Hand.kLeft));
+            }
+
+            @Override
+            protected boolean isFinished() {
+                // Make this return true when this Command no longer needs to run execute()
+                return false;
+            }
+
+            @Override
+            protected void end() {
+                // Called once after isFinished returns true
+            }
+        };
+    }
+
+    public Command goXDistance(double distance) {
+        // The command is named "Sample Command" and requires this subsystem.
+        return new Command("GoXDistance", this) {
+            @Override
+            protected void initialize() {
+                gyroDrivePID.reset();
+                gyroDrivePID.setSetpoint(gyro.getAngle());
+                gyroDrivePID.enable();
+                leftEncoder.reset();
+                rightEncoder.reset();
+            }
+
+            @Override
+            protected void execute() {
+                drive.arcadeDrive(sandstormSpeed, gyroCorrection);
+            }
+
+            @Override
+            protected boolean isFinished() {
+                // Make this return true when this Command no longer needs to run execute()
+                if (leftEncoder.get() + rightEncoder.get() / 2 > distance)
+                    return true;
+                else
+                    return false;
+            }
+
+            @Override
+            protected void end() {
+                gyroDrivePID.disable();
+            }
+        };
+    }
+
+    public Command turnXDegrees(double degrees) {
+        // The command is named "Sample Command" and requires this subsystem.
+        return new Command("turnXDegrees", this) {
+            @Override
+            protected void initialize() {
+                gyroDrivePID.reset();
+                gyroDrivePID.setSetpoint(degrees);
+                gyroDrivePID.enable();
+            }
+
+            @Override
+            protected void execute() {
+                drive.arcadeDrive(0, gyroCorrection);
+            }
+
+            @Override
+            protected boolean isFinished() {
+                // Make this return true when this Command no longer needs to run execute()
+                if (gyro.getAngle() == degrees)
```

Comment:
- Remove comment
- There's a function called `gyroDrivePID.onTarget()`, you should return that instead. That's the purpose of the setpoint